frameworks/keyed/imba/src/main.imba

```diff
@@ -0,0 +1,120 @@
+var A = ["pretty", "large", "big", "small", "tall", "short", "long", "handsome", "plain", "quaint", "clean", "elegant", "easy", "angry", "crazy", "helpful", "mushy", "odd", "unsightly", "adorable", "important", "inexpensive", "cheap", "expensive", "fancy"]
+var C = ["red", "yellow", "blue", "green", "pink", "brown", "purple", "brown", "white", "black", "orange"]
+var N = ["table", "chair", "house", "bbq", "desk", "car", "pony", "cookie", "sandwich", "burger", "pizza", "mouse",  "keyboard"]
+
+tag GlyphIcon < span
```

Comment:
It is better to inline `GlyphIcon` into the `Row`, Imba should automatically detect that it is a static node.

Reply:
Thanks. Done.